lms/static/js/learner_dashboard/views/collection_list_view.js

```diff
@@ -39,8 +43,23 @@
                             childList.push(child.el);
                         }, this);
 
+                        if (this.titleContext){
+                            this.$el.before(HtmlUtils.ensureHtml(this.getTitleHtml()).toString());
+                        }
                         this.$el.html(childList);
                     }
+                },
+
+                getTitleHtml: function(){
+                    var titleHtml = HtmlUtils.joinHtml(
+                        HtmlUtils.HTML('<'), 
+                        this.titleContext.el,
```

Comment:
The only reason would be that ```this.titleContext.el``` is a string defined elsewhere, but I am fine keeping it as it is. Just thought I would mention it.